### ISSUE TITLE
1.20.4? [Suggestion]

### ISSUE THREAD

**Issue by 12Mosch** [purejosh/ancientvessels#2]
Can you update your Mod for **1.20.4 fabric**?

**Comment by purejosh** [comment]
Obviously, this is on my todo list, but there have been some issues with some of the entity related checks I'm doing. Glad to hear that you're using Ancient Vessels, I'm shocked it isn't more popular. I'll do what I can.

**Comment by purejosh** [comment]
I won't be targeting 1.20.4 for my next update. I'm jumping to 1.21.1 next. Apologies.
